github-runner-manager/src/github_runner_manager/platform/jobmanager_provider.py

```diff
@@ -168,56 +174,43 @@ def get_runner_context(
+            updated_metadata.runner_id = str(response.id)
+            if token := response.token:
+                jobmanager_endpoint = f"{self._url}/v1/runners/{updated_metadata.runner_id}/health"
+                # For now, use the first label
+                label = "undefined"
+                if labels:
+                    label = labels[0]
+                command_to_run = (
+                    f"BUILDER_LABEL={label} JOB_MANAGER_BEARER_TOKEN={token} "
+                    f"JOB_MANAGER_API_ENDPOINT={jobmanager_endpoint} "
+                    "builder-agent"
                 )
-                updated_metadata.runner_id = str(response.id)
-                if token := response.token:
-                    jobmanager_endpoint = (
-                        f"{self._url}/v1/runners/{updated_metadata.runner_id}/health"
-                    )
-                    # For now, use the first label
-                    label = "undefined"
-                    if labels:
-                        label = labels[0]
-                    command_to_run = (
-                        f"BUILDER_LABEL={label} JOB_MANAGER_BEARER_TOKEN={token} "
-                        f"JOB_MANAGER_API_ENDPOINT={jobmanager_endpoint} "
-                        "builder-agent"
-                    )
-                    return (
-                        RunnerContext(
-                            shell_run_script=command_to_run,
-                            ingress_tcp_ports=[8080],
+                return (
+                    RunnerContext(
+                        shell_run_script=command_to_run,
+                        ingress_tcp_ports=[8080],
+                    ),
+                    SelfHostedRunner(
+                        identity=RunnerIdentity(
+                            instance_id=instance_id,
+                            metadata=metadata,
                         ),
-                        SelfHostedRunner(
-                            identity=RunnerIdentity(
-                                instance_id=instance_id,
-                                metadata=metadata,
-                            ),
-                            busy=False,
-                            id=int(updated_metadata.runner_id),
-                            labels=[SelfHostedRunnerLabel(name=label) for label in labels],
-                            status=GitHubRunnerStatus.OFFLINE,
-                        ),
-                    )
-                raise PlatformApiError("Empty token from jobmanager API")
-            except (ApiException, RequestError, ValidationError) as exc:
-                logger.exception("Error calling jobmanager api.")
-                raise PlatformApiError("API error") from exc
+                        busy=False,
+                        id=int(updated_metadata.runner_id),
+                        labels=[SelfHostedRunnerLabel(name=label) for label in labels],
```

Comment:
Is the `SelfHostedRunnerLabel` class necessary? The only value inside is the "name" parameter, we should probably just use it as a type, i.e.
`SelfHostedRunnerLabel = str`